### PR TITLE
New BigDecimal test for stripTrailingZeros()

### PR DESCRIPTION
**Pull request by phvega**
Test for ingelabs/classpath#13

### CODE REVIEW

File: src/malva/java/math/BigDecimalTest.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package malva.java.math;
+
+import java.math.BigDecimal;
+
+import malva.TestCase;
+
+public class BigDecimalTest extends TestCase {
+  public static void testStrippingZeros() {
```
Comment (guillerodriguez) [inline src/malva/java/math/BigDecimalTest.java]: Let's following naming conventions and call this `test<NameOfMethodBeingTested>` (see other tests)

File: src/malva/java/math/BigDecimalTest.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    // Array as { test, result }
+    final BigDecimal[][] testCases = {
+      {new BigDecimal("0.1234000"), new BigDecimal("0.1234")},
+      {new BigDecimal("0e2"), BigDecimal.ZERO},
+      {new BigDecimal("100000"), new BigDecimal("1e5")}
```
Comment (guillerodriguez) [inline src/malva/java/math/BigDecimalTest.java]: Better to keep this as multiple assertEquals calls, this makes it easier to find out the offending test cases if a test fails (as line numbers will be included in the stack trace)

**Review by guillerodriguez — CHANGES_REQUESTED** [review]

File: src/malva/java/math/BigDecimalTest.java (hunk 2 above)
Comment (phvega) [reply src/malva/java/math/BigDecimalTest.java]: Fixed

File: src/malva/java/math/BigDecimalTest.java (hunk 1 above)
Comment (phvega) [reply src/malva/java/math/BigDecimalTest.java]: Ok

**Review by guillerodriguez — APPROVED** [review]

**Pull request merged**